lib/parse/data/parseRssItems.ts

```diff
@@ -84,6 +84,30 @@ const parseRssItems = (
           // episode was filtered out. Prepend it to items.
           return [episode, ...items];
         },
+        // Sort Serial feed items.
+        (items: IRssItem[]) =>
+          rssItunesType === 'serial'
+            ? [...items].sort((a, b) => {
+                const {
+                  season: aSeason = '1',
+                  episode: aEpisode,
+                  episodeType: aEpisodeType
+                } = a.itunes;
+                const {
+                  season: bSeason = '1',
+                  episode: bEpisode,
+                  episodeType: bEpisodeType
+                } = b.itunes;
+
+                if (aSeason === bSeason) {
+                  if (aEpisodeType === 'trailer') return -1;
```

Comment:
Okay, so within a single `serial` season, you always put the trailer first.  That seems right.